adminSiteClient/ChartViewIndexPage.tsx

```diff
@@ -0,0 +1,159 @@
+import React, {
+    useCallback,
+    useContext,
+    useEffect,
+    useMemo,
+    useState,
+} from "react"
+import { Button, Flex, Input, Space, Table } from "antd"
+
+import { AdminLayout } from "./AdminLayout.js"
+import { AdminAppContext } from "./AdminAppContext.js"
+import { Timeago } from "./Forms.js"
+import { ColumnsType } from "antd/es/table/InternalTable.js"
+import { ApiChartViewOverview } from "../adminShared/AdminTypes.js"
+import { BAKED_GRAPHER_URL } from "../settings/clientSettings.js"
+import { Link } from "./Link.js"
+import {
+    buildSearchWordsFromSearchString,
+    filterFunctionForSearchWords,
+    highlightFunctionForSearchWords,
+} from "../adminShared/search.js"
+
+function createColumns(
+    highlightFn: (
+        text: string | null | undefined
+    ) => React.ReactElement | string
+): ColumnsType<ApiChartViewOverview> {
+    return [
+        {
+            title: "Preview",
+            key: "chartConfigId",
+            dataIndex: "chartConfigId",
+            width: 200,
+            render: (chartConfigId) => (
+                <img
+                    src={`${BAKED_GRAPHER_URL}/by-uuid/${chartConfigId}.svg`}
```

Comment:
This will work on prod but not on staging servers, right? Could we make this work now that we have CF preview deploys?

Reply:
No, this will also work on staging sites, as long as that config is available.

See http://staging-site-marcel/grapher/by-uuid/0191b6c7-364e-7cf2-be6e-8cee8e073203.svg, for example!